### PR TITLE
Changed "Foreign" to "Non-Latin" and added linux related info

### PR DESCRIPTION
**Pull request by GithubAnon0000**
Fix: #241 

Not sure if adding detailed info regarding input methods is a good idea. It depends heavily on the input methods you use, which also depends on the operating systems.

Currently we list a few sources and even provide a few search prompts to aid the user. From there, they'll probably be able to get the keyboard working or ask in an appropriate forum for their OS.

### CODE REVIEW

File: src/editing.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ reviewing; for that, please see [templates](templates/intro.md).
 
 - **Editing Font** allows you to customize the font and size used when
   editing notes. This is useful if you want to make unimportant
-  information smaller, or increase the size of foreign characters which
+  information smaller, or increase the size of non-english characters which
```
Comment (brishtibheja) [inline src/editing.md:134]:
```suggestion
  information smaller, or increase the size of non-latin characters which
```

File: src/editing.md [hunk 2]
```diff
@@ -488,10 +488,10 @@ can use. Of note:
 While reviewing IO Cards a "Toggle Masks" button will appear just below the image.
 This button will temporary clear all shapes of the note when using "Hide All, Guess One" mode.
 
-## Inputting Foreign Characters and Accents
+## Inputting Non-English Characters and Accents
```
Comment (brishtibheja) [inline src/editing.md:491]:
```suggestion
## Inputting Non-Latin Characters and Accents
```

File: src/editing.md [hunk 3]
```diff
@@ -488,10 +488,10 @@ can use. Of note:
...
 
 All modern computers have built-in support for typing accents and
-foreign characters, and multiple ways to go about it. The method we
+non-english characters, and multiple ways to go about it. The method we
```
Comment (brishtibheja) [inline src/editing.md:494]:
```suggestion
non-latin characters, and multiple ways to go about it. The method we
```

File: src/editing.md [hunk 4]
```diff
@@ -500,17 +500,21 @@ have their own layouts specific to that language.
...
+apostrophe (´) then the letter a (a) gives á.
 
 To add the international keyboard on Windows machines, please see
 <https://thegeekpage.com/how-to-add-us-international-keyboard-in-windows-10/>
 
 To add it on Macs, please see
 <http://www.macworld.com/article/1147039/os-x/accentinput.html>
 
+To add it on Linux, please see
+<https://wiki.archlinux.org/title/Input_method> or <https://wiki.debian.org/Keyboard#Modern_strategy>.
```
Comment (brishtibheja) [inline src/editing.md:512]: Is this not explaining an IME which is different from what the other links are doing here?

We can organise the info better in this topic.

Comment (brishtibheja) [reply src/editing.md]: Looks much better than before.

**Comment by brishtibheja** [comment]
You should've directly added all my suggestions: it's not *non-latin*, it's *non-Latin*.

File: src/editing.md [hunk 5]
```diff
@@ -500,23 +500,40 @@ have their own layouts specific to that language.
...
+- KDE Plasma: <https://userbase.kde.org/Tutorials/ComposeKey>
+
+### Adding keyboard layouts for specific languages
+Keyboards for specific languages are added in a similar way, but we can
 not cover them all here. For more information, please try searching
-Google for "input Japanese on a mac", "type Chinese on Windows 10", and
+on the internet for "input Japanese on a mac", "type Chinese on Windows 10", and
 so on.
 
+For linux it's best to look at the wiki pages of your distro, e.g.
```
Comment (dae) [inline src/editing.md:526]:
```suggestion
For Linux it's best to look at the wiki pages of your distro, e.g.
```

File: src/editing.md [hunk 6]
```diff
@@ -500,23 +500,40 @@ have their own layouts specific to that language.
...
+[arch linux](https://wiki.archlinux.org/title/Input_method) and
```
Comment (dae) [inline src/editing.md:527]:
```suggestion
[Arch Linux](https://wiki.archlinux.org/title/Input_method) and
```

File: src/editing.md [hunk 7]
```diff
@@ -500,23 +500,40 @@ have their own layouts specific to that language.
...
+[debian linux](https://wiki.debian.org/Keyboard#Modern_strategy).
```
Comment (dae) [inline src/editing.md:528]:
```suggestion
[Debian Linux](https://wiki.debian.org/Keyboard#Modern_strategy).
```

File: src/editing.md [hunk 8]
```diff
@@ -500,23 +500,40 @@ have their own layouts specific to that language.
...
+As an example, `apt install ibus-anthy` on Debian allows you to type hiragana characters.
+
+### Right-to-left languages
```
Comment (dae) [inline src/editing.md:531]:
```suggestion
### Right-to-left Languages
```

**Comment by dae** [comment]
Thank you both!

**Pull request merged**